example-server.js

```diff
@@ -7,4 +7,4 @@ server.on('message', function(m) {
 server.on('listening', function(address) {
     console.log('listening on ' + address);
 });
-server.bind();
```

Comment:
There should probably be a comment in here indicating the example server listens on 1067 instead of 67 to avoid having to run it as root. But if you want the example server to print anything it needs to receive requests from an example client on 1067 or you actually do need to run it as root on 67.